config.sample.yml

```diff
@@ -3,6 +3,8 @@ gateway_polling:
   # Flag to enable or disable gateway polling
   enabled: false
   gateway_url: http://ip.or.hostname.of.the.gateway
+  # optional:
```

Comment:
Perhaps something like

> If you have enabled authentication on the gateway, specify the API key (bearer token) from the gateway Access Settings configuration page

would be more descriptive as the comment.

Also it seems that if a gateway is configured with no authentication, the authentication actually fails if you supply any bearer token at all, so the default value in the sample config should be an empty string to avoid confusing authentication failures in case one just copies the entire sample config as-is without planning on using authentication

Reply:
Now the field is still there, but it has the empty value.